.github/workflows/test.yml

```diff
@@ -260,7 +260,7 @@ jobs:
         fetch-depth: 1
     - uses: actions/setup-go@v4
       with:
-        go-version: 1.20.x
+        go-version: 1.21.x
```

Comment:
Is this necessary?

Reply:
Was using clear() which is present from 1.21. I could replace it with Maps.clear as well if we need to keep 1.20

Reply:
Changed code to support 1.20

Reply:
> Changed code to support 1.20

Could you revert the changes to GHA too?

Reply:
Done